### PR TITLE
Using system account PATs with decK

### PR DESCRIPTION
**Pull request by lena-larionova**
### Description

Document system account tokens in decK guides.
Had to split out the include, since we can't use version conditionals in the Konnect docs, but they're needed for decK docs.

Ticket: https://konghq.atlassian.net/browse/DOCU-2838; expanding on info in https://github.com/Kong/docs.konghq.com/pull/4999.  

❗ Merge only when https://github.com/Kong/docs.konghq.com/pull/4999 is merged.

### Testing instructions

Netlify link: 

### Checklist 

- [x] Review label added 
- [x] PR pointed to correct branch (`main` for immediate publishing, or a release branch: e.g. `release/gateway-3.2`, `release/deck-1.17`)

### CODE REVIEW

**Review by cloudjumpercat — APPROVED** [review]
LGTM!

**Review by acgoldsmith — APPROVED** [review]
LGTM - prepping for release by Hayden along w/ [PR4999 ](https://github.com/Kong/docs.konghq.com/pull/4999)

File: app/_includes/md/personal-access-token.md [hunk 1]
```diff
@@ -1,13 +1,36 @@
...
+To generate a PAT for a user account in {{site.konnect_short_name}}, select your name to open the context menu 
+ and click **Personal access tokens**, then click **Generate token**. 
+
+{% endnavtab %}
+{% navtab System Account Token %}
+
+To generate a system account access token, navigate to **Organization > System Accounts** in {{site.konnect_short_name}}, then 
+then click **Generate token**. 
+
+Or, you can [create a token through the Konnect API](/konnect/org-management/system-accounts/#generate-a-system-account-access-token).
```
Comment (lena-larionova) [inline app/_includes/md/personal-access-token.md:30]:
```suggestion
[Create a system account token](/konnect/org-management/system-accounts/#generate-a-system-account-access-token) through the Konnect API.
```

File: app/_src/deck/guides/konnect.md [hunk 2]
```diff
@@ -122,13 +131,57 @@ Successfully Konnected to the Example-Name organization!
...
+To generate a PAT for a user account in {{site.konnect_short_name}}, select your name to open the context menu 
+ and click **Personal access tokens**, then click **Generate token**. 
+
+{% endnavtab %}
+{% navtab System Account Token %}
+
+To generate a system account access token, navigate to **Organization > System Accounts** in {{site.konnect_short_name}}, then 
+then click **Generate token**. 
+
+Or, you can [create a token through the Konnect API](/konnect/org-management/system-accounts/#generate-a-system-account-access-token).
```
Comment (lena-larionova) [inline app/_src/deck/guides/konnect.md:164]:
```suggestion
[Create a system account token](/konnect/org-management/system-accounts/#generate-a-system-account-access-token) through the Konnect API.
```

**Pull request merged**